src/main/daml/ContingentClaims/Core/Observation.daml

```diff
@@ -3,8 +3,8 @@
 
 module ContingentClaims.Core.Observation where
 
-import DA.Foldable (Foldable(..))
-import DA.Traversable (Traversable(..))
+import DA.Foldable (Foldable, foldr)
```

Comment:
Does this mean foldr is now taken from DA.Foldable instead of Prelude? Any difference between the two implementations?